h/static/scripts/test/admin-users-test.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+var AdminUsersController = require('../admin-users');
+
+describe('CreateGroupFormController', function () {
+  var element;
+  var template;
+
+  before(function () {
+    template = '<form id="js-users-delete-form">' +
+               '<input type="submit">';
+  });
+
+  beforeEach(function () {
+    element = document.createElement('div');
+    element.innerHTML = template;
+  });
+
+  it('it should confirm the delete user form submission', function () {
+    // click on submit button
```

Comment:
You can use `element.querySelector()` to get a reference to the button and then just call its `click()` method.